cmd/bundle/init.go

```diff
@@ -32,6 +32,10 @@ var nativeTemplates = []nativeTemplate{
 		name:        "default-python",
 		description: "The default Python template",
 	},
+	{
+		name:        "default-sql",
+		description: "The default SQL template for .sql files executed with Workflows",
```

Comment:
"executed" is one of those grey words that we're trying to get rid off wherever possible. How about "run?"

Also, "Workflows" by itself might be confusing from a branding perspective. How about "Databricks Workflows" (or "Databricks jobs")?

Reply:
Thanks, I'll get rid of "executes"!
```suggestion
		description: "The default SQL template for .sql files that run with Workflows",
```

I'm using just "Workflows" here for symmetry with https://github.com/databricks/cli/pull/1052 where we say "Notebooks / Delta Live Tables / Workflows". I'm not sure we should say "Notebooks / Delta Live Tables / Databricks Workflows" there?